src/collections/pages/fr-CA/about.md

```diff
@@ -0,0 +1,8 @@
+---
+title: À propos
+order: 2
+excerpt: "Ceci est une brève description de la page. "
+---
+Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore
+magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd
+gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
```

Comment:
Similar to other example pages, the about page should have a real description and be localized to the specified language.

Reply:
<insert "It's all Greek to me" joke>